magma/magma
# Prepare folder structure for WebAssembly worker

Summary:
* Move conductor proxy to `workflows/proxy`
* Create  `workflows/wasm-worker` folder structure

Move conductor proxy from `workflows` to `workflows/proxy` subfolder. Create `workflows/wasm-worker` nodejs project and container. Currently it only polls `GLOBAL___js` tasks and marks them as completed immediately.

Test plan:
* Create `GLOBAL___js` taskdef and a workflow as described in the [prototype's readme](https://github.com/tomasol/wasm-conductor-worker-prototype/blob/master/README.md)
* Execute workflow with this task type
* Observe `wasm-worker` logs, it is expected that following lines will appear immediately:
```
GLOBAL___js {
  args: 'q',
  outputIsJson: 'true',
  script: "console.log(JSON.stringify({name: process.argv[1]})); console.error('logging');"
}
GLOBAL___js {
  args: null,
  outputIsJson: 'true',
  script: "console.error('Input', process.argv);let json=JSON.parse(process.argv[1]); json.name_length = json.name.length; console.log(JSON.stringify(json));"
}
```
The tasks should be marked as `COMPLETED` with no output.
